### DISCUSSION TITLE
after entering ' brew install qemu" Error: No similarly named formulae found.

### DISCUSSION THREAD

**Discussion by regularhobby557** [Homebrew/discussions#2190] [Everyday usage]
<img width="560" alt="Screenshot 2021-09-25 at 12 11 22 AM" src="https://user-images.githubusercontent.com/91362739/134760748-92fab8b8-23a4-443b-a51d-f60500acbd39.png">
 
anyone know of a fix

**Comment by carlocab** [comment]
What’s the output of `brew config` and `brew doctor`?

By the way, please [don't use screenshots](https://meta.stackoverflow.com/a/285557) when sharing the output of commands in your terminal. They're often difficult to read and impossible to reuse.
